### ISSUE TITLE
Double line-breaks

### ISSUE THREAD

**Issue by GaiaLembi** [Brown-University-Library/iip-texts#108]
I found double line-breaks in the diplomatic transcription of some inscriptions from Fiq: I canceled them, but it is worthwhile to check if there are more. I surmise that you could then perform a global change to take out the second one

**Comment by emylonas** [comment]
Here are 125 more. I can eliminate automatically, but I wonder if they are standing in for missing lines or vertical spacing?
```
beth0170 <p>1. Παυλίνου<lb/>
               <lb/> 2. Παυ<lb break="no"/>λίνου <lb/> 3.

birs0006 <lb/>
               <lb/>ΚΑΙΛΑΜΠΡΟΙΝΟ<lb/>ΤΑΙΚΑΙΟΙΑΓΙΟΙ<lb/>ΚΟϹΜΟΥΝΤΑΙΚΑΙ<lb/>ΟΛΑΟϹΑΓΑΛΛΕΤS<lb/>ΔΙΑΜΕΙ...

eluy0001 <p>ΘΑΡΣΙ<lb/>
               <lb/> ΑΝΤΙΟΧ<lb/>

eluy0001 <lb/> ΑΝΤΙΟΧ<lb/>
               <lb/> ΑΙΤΩ</p>

eluy0002 <p>ΕΥΜ<lb/>
               <lb/> ΟΙΡΙ<lb/>

eluy0002 <lb/> ΟΙΡΙ<lb/>
               <lb/> ΚΛΕΟ<lb/>

eluy0002 <lb/> ΚΛΕΟ<lb/>
               <lb/> ΠΑΤΡΑ</p>

fiqg0002 <p>SPES<lb/>
               <lb/>

fiqg0002 <p>SPES<lb/>
               <lb/>

fiqg0004 <p>ΕΠΙΤΟΥΘΕΟΣΕΒ<lb/>
               <lb/>

fiqg0004 ΕΠΙΣΚΓΕΡΟΝΤΙ<lb/>
               <lb/>

fiqg0004 ΟΥSΚΑΣΣΙΣΟΥΠ<lb/>
               <lb/>

fiqg0007 ΝΙΩ<lb/>
               <lb/>

fiqg0007 <unclear/>ΤΩΝ<lb/>
               <lb/>

fiqg0008 <p>ΘΑΡΣΕΙ<lb/>
               <lb/>

fiqg0008 <unclear/>ΥΚΥΝ<lb/>
               <lb/>

fiqg0008 <unclear/>ΡΗ<lb/>
               <lb/>

fiqg0009 <p>ΘΕΟ<lb/>
               <lb/>

fiqg0009 ΦΑΝΑ<lb/>
               <lb/>

fiqg0010 <p>ΘΑΡΣ<lb/>
               <lb/>

fiqg0010 ΙΠΕ<lb/>
               <lb/>

fiqg0010 ΤΡΕ<lb/>
               <lb/>

fiqg0011 <p>ΘΑΡΣΙ<lb/>
               <lb/>

fiqg0011 ΜΑΓΝΟΣ<lb/>
               <lb/>

fiqg0011 ΜΟΔΤΑΟΥ<lb/>
               <lb/>

fiqg0012 <p> +<lb/>
               <lb/>

fiqg0012 ΘΑΡΣΙ<lb/>
               <lb/>

fiqg0012 ΝΑΝ<lb/>
               <lb/>

fiqg0012 ΝΑΔΙ<lb/>
               <lb/>

fiqg0013 <lb/>
               <lb/>
fiqg0013 <lb/>
               <lb/>
fiqg0013 ΕΤ<lb/>
               <lb/>

fiqg0014 <lb/>
               <lb/>
fiqg0014 ΕΛΩΟ<lb/>
               <lb/>

fiqg0015 <p>ΘΑΡΣΙ<lb/>
               <lb/>

fiqg0015 ΑΝΤΙΟ<lb/>
               <lb/>

fiqg0015 ΧΕΕΤ<lb/>
               <lb/>

fiqg0016 <p>ΘΑΡΣ<lb/>
               <lb/>

fiqg0016 ΙΑΝΤΙ<lb/>
               <lb/>

fiqg0016 ΟΧΕΠΡ<lb/>
               <lb/>

fiqg0016 ΩΤΕΤ<lb/>
               <lb/>

fiqg0017 <p>ΘΑΡΣΙ<lb/>
               <lb/>

fiqg0017 ΔΩΡ<lb/>
               <lb/>

fiqg0017 ΟΘΕΑ<lb/>
               <lb/>

fiqg0017 ΕΤΩΝ<lb/>
               <lb/>

fiqg0018 <gap reason="lost" extent="unknown" unit="character"/>ΙΕΙ<lb/>
               <lb/>

fiqg0018 <gap reason="lost" unit="character" extent="unknown"/>ΟΝΤΙ<lb/>
               <lb/> ΟΙΘΕΟΙ<lb/>

fiqg0018 <lb/> ΟΙΘΕΟΙ<lb/>
               <lb/> ΑΘΑΝΑΤ<lb/>

fiqg0018 <lb/> ΑΘΑΝΑΤ<lb/>
               <lb/> ΟΙΕΤΙΒ</p>

fiqg0019 <p>ΘΑΡΣΙ<lb/>
               <lb/>

fiqg0019 ΤΑΥΡΙ<lb/>
               <lb/>

fiqg0020 <p>ΘΑΡΣΕ<lb/>
               <lb/>

fiqg0020 ΓΑΙΑΝΕ<lb/>
               <lb/>

fiqg0020 ΕΤΩΝ<lb/>
               <lb/>

fiqg0021 <lb/>
               <lb/>
fiqg0021 </supplied>ΑΒΕΛΟΥΟ<lb/>
               <lb/>

fiqg0021 </supplied>ΑΟΣΜΗΡΡΑ<lb/>
               <lb/>

fiqg0022 <lb/>
               <lb/>
fiqg0022 <lb/>
               <lb/>
fiqg0022 <lb/>
               <lb/>
fiqg0023 <lb/>
               <lb/>
fiqg0023 <lb/>
               <lb/>
fiqg0024 <p>ΔΙΟΚΛΗΤΙΑΝΟΣ<lb/>
               <lb/>

fiqg0024 ΚΑΙΜΑΞΕΙΜΑΝΟΣ<lb/>
               <lb/>

fiqg0024 ΣΕΒΑΣΤΟΙ<lb/>
               <lb/>

fiqg0024 ΚΑΙΚΩΝΣΤΑΝΤΙΟΣ<lb/>
               <lb/>

fiqg0024 ΚΑΙΜΑΞΕΙΜΙΑΝΟΣ<lb/>
               <lb/>

fiqg0025 <p>ΚΑΙΜΑΞΙΜΙΑΝ<lb/>
               <lb/>

fiqg0025 ΣΕΒΒΚΑΙ<lb/>
               <lb/>

fiqg0025 ΚΩΣΤΑΝΤΙΟΣ<lb/>
               <lb/>

fiqg0025 ΚΑΙΜΑΞΙΜΙΑΝ<lb/>
               <lb/>

fiqg0025 ΕΠΙΦΑΝΕΣΤΑΤ<lb/>
               <lb/>

fiqg0025 ΚΑΙΣΑΡΕΣ<lb/>
               <lb/>

fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
fiqg0026 <lb/>
               <lb/>
hamm0044 ΤΟΥ<lb/>
               <lb/>

hamm0044 C<lb/>
               <lb/>

hamm0044 του<lb/>
               <lb/>

hamm0044 ς<lb/>
               <lb/>

hamm0059 <lb/>
                    <lb/>
hamm0069 ΟΝΑΓΑΡ<lb/>
               <lb/>

hamm0069 <lb/>
               <lb/>
idum0552 qabs,<lb/><lb/>two and a half. On the 6th of Shebat.</p>
Match: <lb/><lb/>
mare0205 <p>Δημήτηρ<lb/>μεγάλη<lb/>
                    <lb/>
Match: <lb/>
                    <lb/>
mare0309 <p>Lykos...Zekilios(?)<lb/>
               <lb/>Litarionos</p>

narr0004 <p>ΕΤ + ΕΥΛΗΤΟΣΚΣ ΟΤΙΕΙΣΗΚΟΥΣΕΝ<lb/>
               <lb/>

narr0004 Ο ΤΗΣΦΩΝΗΣ ΤΗΣΔΕΗΣΕ<lb/>
               <lb/>

narr0004 ΙΝ ΩΣΜΟΥΚ <unclear/> ΒΟΗΘΟΣΜ<lb/>
               <lb/>

narr0004 Α ΚΑΙΥΠΕΡ ΑΣΠΙΣΤΗΣ<lb/>
               <lb/>

narr0004 ΜΟΥΕΠΑΥ ΤΩΗΛΠΙΣΑ<lb/>
               <lb/>

narr0005 <p>ΚΕΜΑΣ<lb/>
               <lb/>

narr0005 <unclear/>ΕΚΕΣ<lb/>
               <lb/>

narr0005 ΙΑΖΙΑ<lb/>
               <lb/>

narr0005 <unclear/>Ν<lb/>
               <lb/>

narr0005 <unclear/>ΡΟΥΣ<lb/>
               <lb/>

narr0005 <unclear/>ΚΣ<lb/>
               <lb/>

narr0005 <unclear/>ΗΡΟ<lb/>
               <lb/>

narr0005 ΙΡΟΝΠΙ<lb/>
               <lb/>

narr0005 <unclear/>ΛΣ<lb/>
               <lb/>

narr0005 <lb/>
               <lb/>
susi0002 <p>ΕΠΙΕΥ<lb/>
               <lb/>

susi0004 <p>ΠΡΟΚΟ<lb/>
               <lb/>

susi0004 ΛΟΧΥΨΗ<lb/>
               <lb/>

susi0004 ΤΟCΤΟΝΕ<lb/>
               <lb/>

susi0004 ΠΕΡΜΝ<lb/>
               <lb/>

susi0004 CΕΩCΠΕΤΡΟΥΚ<lb/>
               <lb/>

susi0004 ΝΑCΤΩΝΑΥΤΟΥΥΙ<lb/>
               <lb/>

susi0005 <lb/>
               <lb/>
susi0005 <unclear/>CΤΗΡΙΟΥ<lb/>
               <lb/>

susi0005 ΟΥΑΡΙΩΙ<lb/>
               <lb/>

susi0006 <p>ΠΡΟCΦΟΡΑΟΝΚΥΡΙΟCΓΙΝΟC<lb/>
               <lb/>

susi0006 ΚΗΤΑΟΝΟΜΑΤΑΑΓΙΕΚΩCΜΑ<lb/>
               <lb/>

tibe0004 ...<p>Μάξιμος <lb/> εὐχόμε<lb break="no"/>νος ἐποί<lb break="no"/> ησεν. ζήσῃ.<lb/>
               <lb/>Ἀβουδέ<lb break="no"/>μος εὐχό<lb break="no"/> μενος ἐπ<lb break="no"/>οίησεν...

tibe0004 ...βουδέ<lb break="no"/>μος εὐχό<lb break="no"/> μενος ἐπ<lb break="no"/>οίησεν.<lb/>
               <lb/>Ζώϊλος <lb/>εὐχόμε<lb break="no"/>νος ἐποίη<lb break="no"/>σεν. ζήσῃ.<lb/>

tibe0004 ...<lb/>Ζώϊλος <lb/>εὐχόμε<lb break="no"/>νος ἐποίη<lb break="no"/>σεν. ζήσῃ.<lb/>
               <lb/>Καλίνικο<lb break="no"/>ς εὐχόμε<lb break="no"/>νος ἐποίη<lb break="no"/>σεν....

tibe0004 ...κο<lb break="no"/>ς εὐχόμε<lb break="no"/>νος ἐποίη<lb break="no"/>σεν. ζήσῃ.<lb/>
               <lb/>Σιόρτασις<lb/>

tibe0004 </choice> ἐποίησεν<lb/> σωζέστω.<lb/>
               <lb/>

tibe0004 <lb break="no"/>ενος ἐποίη<lb break="no"/>σεν. ζήσῃ.<lb/>
               <lb/>Σευῆ<supplied reason="undefined">ρος</supplied>

tibe0004 εὐλογία<lb/>αὐτῷ. ἀμήν.<lb/>
               <lb/>Ἰοῦλλος<lb/>προνούμ<lb break="no"/>ενος πάντ<lb break="no"/>α ἐτελίωσε.<lb/>

tibe0004 ...<lb/>Ἰοῦλλος<lb/>προνούμ<lb break="no"/>ενος πάντ<lb break="no"/>α ἐτελίωσε.<lb/>
               <lb/>
```

**Comment by GaiaLembi** [comment]
@emylonas In the inscriptions from Fiq, Narran and Susita they are for sure doubles: the ones I have already corrected come from the same volume -- in this case as well: old inscriptions badly encoded. In some cases missing pieces of information (e.g. letters' height) are actually in the text and have not been put in the files. There should be a headbang emoticon for Github comments too...

**Comment by emylonas** [comment]
😭 😬 😡 😱 

Ok, I'll fix those

**Comment by emylonas** [comment]
closed with commit 91394816a
